src/test/java/Homework21.java

```diff
@@ -15,13 +16,12 @@ public void renamePlaylist() throws InterruptedException {
         playlistPage.addPlaylistName(newPlaylist);
         playlistPage.clickEnter();
         playlistPage.assertPlaylistNameInHeader(newPlaylist);
-        assertSuccessBanner();
+        BasePage.assertSuccessBanner();
 
         playlistPage.rightClickOnPlaylist();
         playlistPage.clickEdit();
         playlistPage.renamePlaylistName(newPlaylist);
-        assertSuccessBanner();
-        driver.navigate().refresh();
+        BasePage.assertSuccessBanner();
         Thread.sleep(4000);
```

Comment:
Please, delete `throws InterruptedException`

**Thread. sleep is bad!** It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.